website/docs/r/config_authorization.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "aws"
+page_title: "AWS: aws_config_authorization"
+sidebar_current: "docs-aws-resource-config-authorization"
+description: |-
+  Provides an AWS Config Authorization.
+---
+
+# aws_config_authorization
+
+Provides an AWS Config Authorization
+
+## Example Usage
+
+```hcl
+resource "aws_config_authorization" "example" {
+  account_id = "123456789012" # Required
+  region     = "eu-west-2"    # Required
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `account_id` - (Required) Account ID
+* `region` - (Required) Region
+
+## Attributes Reference
+
+The following attributes are exported:
```

Comment:
We're in the process of updating all these for clarity (#4685): `In addition to all arguments above, the following attributes are exported:`